### PR TITLE
Re-add DockerHub docs to the CI process.

### PR DESCRIPTION
**Pull request by dchw**
Now that permissions have been sorted, we can re-add this. I have tested the action locally and successfully updated the descriptions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -648,6 +648,22 @@ jobs:
         run: ./build/linux/amd64/earthly config global.local_registry_host 'tcp://127.0.0.1:8371'
       - name: Build and push +prerelease
         run: ./build/linux/amd64/earthly --ci --push +prerelease
+      - name: Update DockerHub description for earthly/earthly
+        uses: peter-evans/dockerhub-description@v2
```
Comment (alexcb) [inline .github/workflows/ci.yml:652]: Can we trust Peter?

Comment (alexcb) [reply .github/workflows/ci.yml:652]: is there a way to pin it to a git sha to avoid it from changing?

Comment (alexcb) [reply .github/workflows/ci.yml:652]: or alternative, we could fork this action to prevent it from changing.

Comment (dchw) [reply .github/workflows/ci.yml:652]: This action appears to be widely used; and what I've seen of the code when I was poking around it looked clean. I'll look to see if we can pin it? I don't think there is significant reason to worry though.

Comment (alexcb) [reply .github/workflows/ci.yml]: https://julienrenaux.fr/2019/12/20/github-actions-security-risk/ makes some good points about why we should create a fork (to protect us from the repo being compromised or deleted).

Comment (dchw) [reply .github/workflows/ci.yml]: I would rather just script it than maintain a fork. Ill switch it to a hand-rolled script then? Or is the SHA pinning enough.

Do we have other actions we should be forking too that I could handle at the same time?

Comment (alexcb) [reply .github/workflows/ci.yml]: pinning a sha1 is sufficient to prevent the issue of the repo being compromised.

Comment (dchw) [reply .github/workflows/ci.yml]: Latest commit should have it pinned.

**Review by alexcb — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (alexcb) [reply .github/workflows/ci.yml]: > Do we have other actions we should be forking too that I could handle at the same time?

Aside from the official github actions, we also use `docker/setup-qemu-action@v1` but it doesn't make use of credentials.

**Pull request merged**